app/lib/geordi-logger.coffee

```diff
@@ -13,9 +13,15 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
   instance: =>
     @geordi = @geordi || @makeGeordi @keys?.projectToken
 
+  getEnv: ->
+    shell_env = process.env.NODE_ENV if process.env.NODE_ENV=="production"
+    reg = /\W?env=(\w+)/
+    browser_env = window?.location?.search?.match(reg)?[1]
+    shell_env || browser_env || 'staging'
```

Comment:
I know. But I don't. I want shell to take priority if shell env=production, otherwise browser params take priority. If shell env not set to production, and browser env not set, use staging.